operator/pkg/cli/upgrade.go

```diff
@@ -74,6 +75,10 @@ func UpgradeCmd() *cobra.Command {
 
 	// TODO(upgrade): local-artifact-mirror-image should be included in the installation object
 	cmd.Flags().StringVar(&localArtifactMirrorImage, "local-artifact-mirror-image", "", "Local artifact mirror image")
```

Comment:
If we were using the cli binary rather than the operator im not sure any of these flags would be required and this would be more straightforward